### DISCUSSION TITLE
MDX support

### DISCUSSION THREAD

**Discussion by tonyxiao** [flowershow/flowershow#291] [General and Help] [answered]
Can I use mdx files with flowershow?

**Comment by tonyxiao** [comment, marked as answer]
Ah never mind just found this in the docs! https://flowershow.app/docs/mdx

**Reply by rufuspollock** [reply]
Great you found it and good question.

@olayway @PhilippeduPreez  this makes me think we should explicitly mention MDX on front page somewhere perhaps in the feature section. Perhaps to start with in "Customize and Extend" or as its own major feature e.g. "Extend markdown with rich components with MDX" (or even more generally "Markdown eXtended" with a sense of what you can do with markdown (in a way taking to publishing what obsidian does to markdown as an editor experience).
